src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        int randomIndex = random.nextInt(Color.values().length);
+        return Color.values()[randomIndex];
+    }
```

Comment:
return String please :)

Reply:
I don't see any sense of refactoring that. We are getting randomColor, which returns us Color, which is supposed to be enum as per task description. We can cast it to String if required or reuse in any other places in future if required